src/main/sensors/pitotmeter.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define PITOT_SAMPLE_COUNT_MAX   48
+
+typedef struct pitotmeterConfig_s {
+    uint8_t pitot_sample_count;              // size of pitot filter array
+    float pitot_noise_lpf;                   // additional LPF to reduce pitot noise
```

Comment:
I think `pitot_sample_count` and `pitot_noise_lpf` are redundant. Pitot is sampled at 100Hz rate, at that rate average of 21 sample is effectively a 5Hz low-pass filter.